apps/mobile/scripts/configureApplication.js

```diff
@@ -0,0 +1,190 @@
+// @flow strict
+
```

Comment:
Can you write some comment or add it to readme why we need this script, please? thanks

Reply:
Sure, I'll add some comments.